buildDockerfile.sh

```diff
@@ -0,0 +1,340 @@
+#!/bin/bash
+#
+# Script to build a docker image that is a Jenkins agent containing the rtc2git migration code.
+#
+# Requires various variables to be set in addition to the arguments below:
+#   $1 = Name of this docker image, e.g. foobar:mybranch-123 or foobar:123
+#
+set -o pipefail
+set -eu
+
+
+
+# Download the internal certificate CA(s) that we need our image to "trust" so that it can
+# connect to our Jenkins servers.
+downloadCertificatesThatOurJvmMustTrust() {
+  mkdir -p TrustedCertificates
+  (
+  cd TrustedCertificates
+  for URL in \
+      'https://i2group.jfrog.io/artifactory/i2eng-generic-local/certs/ibm-internal/ca/intermediate/caintermediatecert.der' \
+      'https://i2group.jfrog.io/artifactory/i2eng-generic-local/certs/ibm-internal/ca/root/carootcert.der' \
```

Comment:
I think you have here some internal links, which you probably didnt want to share? 🙈

Reply:
You're half right.
Yes, they're internal URLs.
No, it doesn't matter that they're shared ... and even if you got a copy, it'd just allow you to trust IBM-internal servers (whose certificates are signed by an IBM-internal CA) 😉 

However, they probably should be abstracted out and replaced with some sort of "put your own CA certs here" placeholder (and make it optional).

Reply:
@WtfJoke @pjdarton as we use IBM parts anyway and this are the CA I see no bigger problems with those...

Reply:
Yup, it's not a confidentiality issue here, it's a code-clarity issue.
I needed to put in some CAs into my docker image but that's not something everyone will need.
...so I've commented that code out and put in a comment explaining what to do if you _do_ need custom CAs added.